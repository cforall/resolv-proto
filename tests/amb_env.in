// Detects failures due to resolution over invalid environment in 
// ambiguous expressions

#a make
#b make

T bind T

#a amb #a
#a amb #b

use #a
%%
use( amb( bind( make() ) ) )
