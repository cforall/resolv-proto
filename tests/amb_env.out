
ERROR: ambiguous resolution for amb( bind{T.2}( make( ) ) )
